cpython/Setup.local

```diff
@@ -35,6 +35,12 @@ _sha3 _sha3/sha3module.c
 _md5 md5module.c
 _blake2 _blake2/blake2module.c _blake2/blake2b_impl.c ../../host/Python-3.7.0/Modules/_blake2/blake2s_impl.c
 
+_sqlite3 _sqlite/cache.c _sqlite/connection.c _sqlite/cursor.c _sqlite/microprotocols.c _sqlite/module.c _sqlite/prepare_protocol.c _sqlite/row.c _sqlite/statement.c _sqlite/util.c -I$(SQLITEBUILD) -L$(SQLITEBUILD) -lsqlite3
+
+_bz2 _bz2module.c -I$(BZIP2BUILD) -L$(BZIP2BUILD) -lbz2
+mmap mmapmodule.c
```

Comment:
I'm kind of amazed this worked out-of-the-box.

Reply:
Me too! There is a Cpython `test_os` that used to fail due to mmap I think. @msabramo what happens if you remove the `mmap` tag in ` test/python_tests.txt` for it?

Reply:
Resolving this because `mmap` was removed from this PR as it was done in #468.